public class practice4 {
    public static void main(String args[]) {
        throw new ArithmeticException("exception caught");
    }
}
